packages/lit-dev-content/site/tutorials/content/working-with-lists/03.md

```diff
@@ -0,0 +1,29 @@
+The rendering methods we've seen so far make it so that Lit keeps track of the
+DOM elements and will update them accordingly when the property items change.
+This is usually fine and efficient but sometimes we may want to control how
+the each item is tied to the DOM element so that we'd rather have the DOM
+element move rather than updating.
+
+For instance, take a look at the example component we have here where each
+todo item is rendered as a list item with a checkbox. We also have buttons
+to sort these items in ascending and descending alphabetical order. Try
+checking one of the items and changing the sort order. You'll notice that
+the item text changes but not the position of the checked box. This is
+because Lit is updating the text rather than moving the entire DOM node.
```

Comment:
I feel like this is missing some more exposition on the underlying reason, but I'd trust someone on the core team to provide better wording than I

cc: @justinfagnani @sorvell @kevinpschaaf